core/src/rust/filodb_core/src/profile.rs

```diff
@@ -0,0 +1,73 @@
+//! Helpers for profiling / testing
+
+#[cfg(feature = "dhat-heap")]
+use std::sync::Mutex;
+
+use jni::{
+    objects::JClass,
+    sys::{jlong, jstring},
+    JNIEnv,
+};
+
+use crate::{exec::jni_exec, state::IndexHandle};
+
+/// Get cache stats info
+#[no_mangle]
+pub extern "system" fn Java_filodb_core_memstore_TantivyNativeMethods_00024_dumpCacheStats(
+    mut env: JNIEnv,
+    _class: JClass,
+    handle: jlong,
+) -> jstring {
+    jni_exec(&mut env, |env| {
+        let index = IndexHandle::get_ref_from_handle(handle);
+
+        let (column_hits, column_misses) = index.column_cache.stats();
+        let (query_hits, query_misses) = index.query_cache_stats();
```

Comment:
The library I'm using doesn't appear to expose any kind of stats reset method.  However, it doesn't matter too much at the moment because the only place that is using this is the JMH tests and they drop the whole JVM after every test anyway.